slides/release-notes/release-notes-2021/aspose-slides-cloud-21-1-release-notes/_index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Aspose.Slides Cloud 21.1 Release Notes"
+type: docs
+url: /aspose-slides-cloud-21-1-release-notes/
+weight: 60
+---
+
+## **Important Changes and Enhancements**
+- SLIDESCLOUD-1066 - Enable PDF Import
+- SLIDESCLOUD-939 - Support ProtectionManager properties
+- SLIDESCLOUD-1073 - Remove password field from request query string
+
+## **Other improvements and changes**
+- SLIDESCLOUD-1026 - NullReferenceException is thrown when not all shape properties are provided
+- SLIDESCLOUD-1036 - Export to SVG with VectorizeText option fails in linux image
+
+## **Public API changes**
+### **PDF import**
+The new **fromHtml** POST method allows to create presentations or append slides to existing ones using PDF files.
```

Comment:
The new **fromHtml** - looks like a copy/paste mistake. The fromPdf method should be used here.